build.gradle

```diff
@@ -54,17 +55,17 @@ compileJava {
 }
 
 dependencies {
-    compile('org.seleniumhq.selenium:selenium-java:3.4.0'){
+    compile('org.seleniumhq.selenium:selenium-java:3.5.1'){
```

Comment:
I've almost created the same. Although having Selenium version assigned to a single variable looks more elegant to me ;)